contracts/.solhint.json

```diff
@@ -25,6 +23,7 @@
     "visibility-modifier-order": "error",
     "reason-string": "error",
     "gas-custom-errors": "off",
-    "max-states-count": ["error", 20]
+    "max-states-count": ["error", 20],
+    "import-path-check": "off"
```

Comment:
Not needed for our path imports, for example import like this is not passing lint: 
`
import {Ownable2StepUpgradeable} from "@openzeppelin/contracts-upgradeable/access/Ownable2StepUpgradeable.sol";
`